src/ContaoManager/Plugin.php

```diff
@@ -28,8 +31,8 @@ class Plugin implements BundlePluginInterface
     public function getBundles(ParserInterface $parser)
     {
         return [
-            BundleConfig::create('Craffft\CssStyleSelectorBundle\CraffftCssStyleSelectorBundle')
-                ->setLoadAfter(['Contao\CoreBundle\ContaoCoreBundle'])
+            BundleConfig::create(CraffftCssStyleSelectorBundle::class)
+                ->setLoadAfter([ContaoCoreBundle::class, RockSolidCustomElementsBundle::class])
```

Comment:
And here the same.

Reply:
::class statements do not need the class to be loaded